frontend/src/Views/MainPage/MainPage.tsx

```diff
@@ -173,12 +176,28 @@ const MainPage: FC = () => {
    * @param     newDetails => A new 'FarmDetailsInterface' object with new data from
    *            from a form section.
    * */
-  const updateFarmDetails = (newDetails: FarmDetailsInterface) => {
+  const updateFarmDetails = (newDetails: FarmDetailsInterface): void => {
     setFarmDetails(newDetails);
     updateLocalDetails(newDetails);
     handleFormState(CmdOptions.FORWARDS, undefined, COMPLETED);
   };
-
+  /**
+   * @summary   Handler for updating the Nutrients Data of the Calculator.
+   * @desc      This updates the temporaryStorage for nutrient data before it gets assigned
+   *            to the farmInformation in the Calculation Module
+   * @param     newNutrients => A new 'TempNutrientInterface' object with new data from
+   *            from the nutrient module.
+   * */
+  const updateNutrientDetails = (newNutrients: TempNutrientsInterface): void => {
+    if (!nutrientsSliceFirstUpdate) {
+      nutrientsSliceFirstUpdate = true;
+      setNutrientDetails((prevDetails) => [...prevDetails.slice(1), newNutrients]);
+    } else {
+      setNutrientDetails((prevDetails) => [...prevDetails, newNutrients]);
+    }
+  };
+  // Development Log
+  console.log(nutrientDetails);
```

Comment:
Is this staying here?

Reply:
This is staying until I make the List. This is for the purpose of proving that I am able to pass in the data from the module into the parent component.